core-build/gulp-core-build-webpack/package.json

```diff
@@ -20,14 +20,15 @@
     "@types/node": "6.0.62",
     "gulp": "~3.9.1",
     "gulp-util": "~3.0.7",
-    "webpack": "~1.13.0"
+    "webpack": "~3.5.5",
+    "ajv": "^5.2.2"
```

Comment:
Adding this here because rush generate barfs without it. Something requires it as a peer dependency.